client/src/project/Form.vue

```diff
@@ -429,13 +430,17 @@
             verified: 1,
             deleted: 0,
             stage: { ne$: 4 },
-            fields$: ['id', 'name'],
+            fields$: ['id', 'name', 'place'],
             sort$: {
               name: 1,
             },
           },
         })).body;
       },
+      getCity(dojos, id) {
+        const result = dojos.find(dojo => dojo.id === id);
```

Comment:
you don't need to pass down dojos, `this.dojos` is accessible.
Doesn't require a change, just an info :)